src/edit/SproutPlayer.vue

```diff
@@ -20,6 +21,7 @@ export default {
   },
   mounted() {
     this.appendVideo();
+    this.$elementBus.on('reload', this.appendVideo);
```

Comment:
Does it make sense to rename the `appendVideo` since it doesn't really append, but rather sets/replaces embeded video?